components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/java/org/wso2/carbon/identity/webhook/metadata/api/constant/ErrorMessage.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.webhook.metadata.api.constant;
+
+/**
+ * Error messages for webhook metadata.
+ */
+public enum ErrorMessage {
+
+    // Client errors (61xxx range)
+    ERROR_CODE_PROFILE_NOT_FOUND("61001", "Profile not found",
```

Comment:
Now my question is from where this error code gets prefixed ?
As per current practice, this should be something like WEBHOOKMETA-61001

If this error does not go to the api level, we can omit the error code fully as well, so it get handled at service later and that adds a proper error code

Reply:
we are appedning it form the api layer
errorCode = errorCode.contains(ERROR_CODE_DELIMITER) ? errorCode :
                ERROR_CODE_PREFIX + errorCode;

Reply:
Additionally, the reason I have added the prefix previously is that this is a service which can be used for service-to-service communication as well. Therefore, having the prefix in the service layer will provide much more context.